option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message FriendRecvEnergyAnswer
{
 required uint32 result = 1;
 repeated U64Type playerid = 2;
 required uint32 energy_recv = 3;
}
